source/rhoas/src/main/java/com/openshift/cloud/beans/KafkaK8sClients.java

```diff
@@ -136,11 +136,8 @@ private CustomResourceDefinition initCloudServiceAccountRequestCRDAndClient(
 
     CustomResourceDefinition akcCrd;
 
-    var crdsItems = crds.customResourceDefinitions().list().getItems();
-    var kafkaConnectionCRDName = CustomResource.getCRDName(CloudServiceAccountRequest.class);
-
-    var akcCrdOptional = crdsItems.stream()
-        .filter(crd -> kafkaConnectionCRDName.equals(crd.getMetadata().getName())).findFirst();
+    var akcCrdOptional = Optional.ofNullable(crds.customResourceDefinitions()
```

Comment:
FYI This whole code can be considered dead/development only and can be removed. CRD's are introduced by OLM so this code would be only for local dev etc.

Reply:
Thanks @wtrocki 👍